docs/dev/Lua API.rst

```diff
@@ -4817,6 +4817,7 @@ cycle through by pressing a specified hotkey or clicking on the text.
 It has the following attributes:
 
 :key: The hotkey keycode to display, e.g. ``'CUSTOM_A'``.
+:key_back: Similar to ``key``, but will cycle backwards (optional)
```

Comment:
also please document the new optional parameter to `cycle()` on line 4838 below

Reply:
Done